backtrace-library/src/main/java/backtraceio/library/metrics/EventsResult.java

```diff
@@ -0,0 +1,98 @@
+package backtraceio.library.metrics;
+
+import com.google.gson.annotations.SerializedName;
+
+import backtraceio.library.models.types.BacktraceResultStatus;
+
+/**
+ * Metrics events send method result
+ */
+public class EventsResult {
+
+    /**
+     * Object identifier
+     */
+    @SerializedName("_rxid")
+    @SuppressWarnings({"UnusedDeclaration"})
+    public String rxId;
```

Comment:
Don't think it actually returns this? Only applicable for reports, not events

Reply:
Yes this parameter is not needed, thank you @vlussenburg !